app/src/main/java/com/pvp/app/service/NotificationReceiver.kt

```diff
@@ -0,0 +1,76 @@
+package com.pvp.app.service
+
+import android.Manifest
+import android.app.AlarmManager
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.pvp.app.R
+
+class NotificationReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent) {
+        showNotification(
+            context,
+            intent.getStringExtra("notificationText"),
+            intent.getIntExtra("notificationId", 0)
+        )
+    }
+
+    private fun showNotification(context: Context, text: String?, notificationId: Int) {
+        val notificationManager = NotificationManagerCompat.from(context)
+        val channelId = "Task"
```

Comment:
If this is channel id and it must be the same for all tasks, I would suggest using `Configuration` interface for this to have a centralized place for constants/configurations. Define something like `val channelNotificationTasksReminderId`.